src/OrleansCodeGenerator/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.0.0-rc2" targetFramework="net45" />
+  <package id="Microsoft.CodeAnalysis.Common" version="1.0.0-rc2" targetFramework="net45" />
+  <package id="Microsoft.CodeAnalysis.CSharp" version="1.0.0-rc2" targetFramework="net45" />
+  <package id="System.Collections.Immutable" version="1.1.33-beta" targetFramework="net45" />
+  <package id="System.Reflection.Metadata" version="1.0.18-beta" targetFramework="net45" />
```

Comment:
I see System.Collections.Immutable 1.1.36 published and System.Reflection.Metadata 1.0.21, with no -beta or -rc. The others are currently -rc3. I presume they will RTM with VS 2015.